client/src/components/ManageAccount/ManageAccount.scss

```diff
@@ -127,9 +140,10 @@
 }
 // Buttons and Forms
 .manage-account-submit-button {
+  // float: right;
   display: flex;
-  justify-content: space-between;
-  align-items: center;
+  width: fit-content;
+  margin-left: auto;
```

Comment:
I think we can lose this line, since we only want to apply the left margin within a modal
```suggestion
```